src/librustc_mir/interpret/validity.rs

```diff
@@ -66,6 +66,7 @@ macro_rules! try_validation {
 pub enum PathElem {
     Field(Symbol),
     Variant(Symbol),
+    GeneratoreState(VariantIdx),
```

Comment:
sp: `GeneratorState`

Reply:
*oopsie* fixed in https://github.com/rust-lang/rust/pull/60569